TECHNICAL_DETAILS.md

```diff
@@ -1,5 +1,29 @@
 # Technical Details #
 
+## Summary of supported features & options ##
+
+| Option  | Status | Override via ASG conf |
+| ------- | :----: | :-------------------: |
+|Run on multiple regions | :white_check_mark: | :heavy_minus_sign: |
+|Keep a fixed minimum percentage of on-demand | :white_check_mark: | :white_check_mark: |
+|Keep a fixed minimum number of on-demand | :white_check_mark: | :white_check_mark: |
+|Can bid at a certain percentage of the on-demand price | :white_check_mark: | :white_check_mark: |
+|Can bid at a higher spot's price | :white_check_mark: | :white_check_mark: |
```

Comment:
I'd rephrase this as something like "Can bid the current spot price plus a certain percentage"